packages/svelte/tests/migrate/samples/is-not-where-has/input.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+	function is(){}
+	function where(){}
+	function not(){}
+	function has(){}
+
+	// looks like css but it's not in style tag
+	const x = {
+		div:is(42),
+		span:where(42),
+		form:not(42),
+		input:has(42),
+	}
+</script>
+
+what if i'm talking about `:has()` in my blog?
+
+```css
+	:has(.is_cool)
+```
+
+<style lang="postcss">
+	div:has(span){}
+	div > :not(span){}
+	div > :is(span){}
+	div > :where(span){}
+
+	div:has(:is(span)){}
+	div > :not(:is(span)){}
+	div > :is(:is(span)){}
+	div > :where(:is(span)){}
```

Comment:
just for safety: can we add more tests around nested braces? Like `:has(.x:not(...))`

Reply:
Sure...if you can think of any other case please let me know

Reply:
`div :is(.class:is(span:is(:hover)), .x){}` -> the `.x` isn't properly wrapped

Reply:
Oh I know why...let me fix